x/treasury/handler.go

```diff
@@ -82,12 +103,19 @@ func handleMsgOracleBurn(ctx sdk.Context, k keeper.Keeper, msg types.MsgOracleBu
 		return err.Result()
 	}
 
-	ctx.EventManager().EmitEvent(
+	ctx.EventManager().EmitEvents(sdk.Events{
+		sdk.NewEvent(
+			types.EventTypeMsgSend,
```

Comment:
This should be a new event type `EventTypeOracleBurn`

Reply:
ok. Modified the code accordingly